### PR TITLE
AWS ECS with terraform

### PR DESCRIPTION
**Pull request by delfrrr**

### CODE REVIEW

File: install/ecs/variables.tf [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+# general
+variable "region" {
+  default = "us-east-1"
```
Comment (denplis) [inline install/ecs/variables.tf:4]: Are you sure about region? Isn't eu-central-1 ?

File: install/ecs/rds.tf [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+resource "aws_db_subnet_group" "dekart_rds" {
+  name       = "${var.dekart_deployment_name}-rds"
+  subnet_ids = aws_subnet.private.*.id
+}
+
+# rds
+
+resource "aws_db_instance" "dekart" {
+  identifier                  = var.dekart_deployment_name
+  allocated_storage           = 10
```
Comment (denplis) [inline install/ecs/rds.tf:34]: as I remember minimum allocated storage is 20gb

File: install/ecs/roles.tf [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+          Effect = "Allow",
+          Action = [
+            "athena:CancelQueryExecution",
+            "athena:Get*",
+            "athena:StartQueryExecution",
+            "athena:StopQueryExecution",
+            "glue:Get*",
+          ],
+          Resource = [
+            "*"
```
Comment (denplis) [inline install/ecs/roles.tf:43]: I Would suggest to list here your resources and not use *, just security note

File: install/ecs/main.tf [hunk 4]
```diff
@@ -0,0 +1,13 @@
+terraform {
```
Comment (denplis) [inline install/ecs/main.tf:1]: can't see terraform backend for keeping state

File: install/ecs/rds.tf (hunk 2 above)
Comment (delfrrr) [reply install/ecs/rds.tf:34]: Hard to say it's from example https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance

File: install/ecs/main.tf (hunk 4 above)
Comment (delfrrr) [reply install/ecs/main.tf:1]: This deployment example, will add comment

File: install/ecs/rds.tf (hunk 2 above)
Comment (delfrrr) [reply install/ecs/rds.tf:34]: You are actually right https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html

**Pull request merged**